src/main/java/com/jcabi/w3c/DefaultValidationResponse.java

```diff
@@ -71,14 +71,12 @@
     /**
      * Set of errors found.
      */
-    private final transient Set<Defect> ierrors =
-        Collections.synchronizedSet(new HashSet<Defect>(0));
+    private final transient Set<Defect> ierrors = new HashSet<Defect>(0);
```

Comment:
@piddubnyi why we can't use CopyOnWriteArraySet here?
I think this collection isn't heavily used, up to a thousand elements
and number of reads should be higher than number of writes